ui/documents/semantic-ui/templates/documents/form/FormFieldsContainer.jsx

```diff
@@ -37,6 +39,12 @@ const FormFieldsContainer = () => {
     []
   );
 
+  const { values, setFieldValue, setFieldTouched } = useFormikContext();
+
+  const validTags =
+    "a,abbr,acronym,b,blockquote,br,code,div,table,tbody,td,th,tr,em,h1,h2,h3,h4,h5,i,li,ol,p,pre,span,strike,strong,sub,sup,u,ul";
```

Comment:
Is there any difference between this and [this](https://github.com/oarepo/oarepo-ui/pull/188/files#diff-7faf2dfbd5b15c265ec6341bbbee21521b1d5faaea7e871e5036f06fdc6f5f88R30) ?

Reply:
Perfect, not it's exactly the same

Reply:
Now what is the added value of exactly duplicating this set of tags, if it's the default value for sanitize function?

Reply:
Tiny-mce expects it in the different format than we have it for sanitisation function